tratto/src/main/java/star/tratto/identifiers/JPType.java

```diff
@@ -0,0 +1,65 @@
+package star.tratto.identifiers;
+
+import com.github.javaparser.ast.type.Type;
+
+public enum JPType {
+    ARRAY_TYPE("ARRAY_TYPE"),
+    CLASS_OR_INTERFACE_TYPE("CLASS_OR_INTERFACE_TYPE"),
+    INTERSECTION_TYPE("INTERSECTION_TYPE"),
+    PRIMITIVE_TYPE("PRIMITIVE_TYPE"),
+    REFERENCE_TYPE("REFERENCE_TYPE"),
+    TYPE_PARAMETER("TYPE_PARAMETER"),
+    UNION_TYPE("UNION_TYPE"),
+    UNKNOWN_TYPE("UNKNOWN_TYPE"),
+    VAR_TYPE("VAR_TYPE"),
+    VOID_TYPE("VOID_TYPE"),
+    WILDCARD("WILDCARD");
+
+    private final String type;
```

Comment:
Is this variable needed?  It seems to be the default that is used by the value and valueOf methods, so the code would be smaller without it.